app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt

```diff
@@ -250,11 +244,10 @@ fun RunHikeBottomSheet(
                       style = MaterialTheme.typography.bodyLarge,
                       fontWeight = FontWeight.Bold,
                       textAlign = TextAlign.Right,
-                      modifier = Modifier.padding(top = 8.dp).testTag(TEST_TAG_TOTAL_DISTANCE_TEXT),
+                      modifier = Modifier.testTag(RunHikeScreen.TEST_TAG_TOTAL_DISTANCE_TEXT),
                   )
                 }
 
-            val elevationGainString = hikeRoute.elevationGain.roundToInt().toString()
             val hourString =
                 String.format(Locale.getDefault(), "%02d", (hikeRoute.estimatedTime / 60).toInt())
             val minuteString =
```

Comment:
Change: these variables are now unused, my suggestion would be to create `val hours` and `val minutes` and use them below instead of re-computing every time.

```suggestion
            val hours = hikeRoute.estimatedTime / 60
            val minutes = hikeRoute.estimatedTime % 60
```

Reply:
done